clang/test/CXX/drs/dr23xx.cpp

```diff
@@ -182,8 +182,8 @@ struct Bad2 { int a, b; };
 } // namespace dr2386
 namespace std {
 template <typename T> struct tuple_size;
-template <> struct std::tuple_size<dr2386::Bad1> {};
```

Comment:
Is this a really necessary change?

Reply:
@Endilll if left unchanged this will result in a warning about redundant qualification, which I don't think is the point of the test... alternatively I can add `// expected-warning {{extra qualification on member}}`

Reply:
Is this warning emitted if we spell it `::std::tuple)size`?

Reply:
It is, yes